### PR TITLE
Makes Plasma 5 is default desktop

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

File: control/control.openSUSE.xml [hunk 1]
```diff
@@ -301,6 +300,13 @@ textdomain="control"
                 </window_manager>
 
                 <window_manager>
+                    <sysconfig_wm>startkde</sysconfig_wm>
```
Comment (jreidinger) [inline control/control.openSUSE.xml]: this looks strange to have two time here `startkde` I think it should be only once there. So remove old one or choose new sysconfig_wm for plasma.

File: control/control.openSUSE.xml [hunk 2]
```diff
@@ -385,7 +391,7 @@ is the most appropriate desktop for you.</label></desktop_dialog>
 
         <!-- Desktop dialog: desktop names -->
         <desktop_gnome><label>GNOME Desktop</label></desktop_gnome>
-        <desktop_kde><label>KDE Desktop</label></desktop_kde>
+        <desktop_kde><label>KDE Plasma 5 Desktop</label></desktop_kde>
```
Comment (jreidinger) [inline control/control.openSUSE.xml]: What is difference? Why not keep simple KDE?

**Comment by jreidinger** [comment]
LGTM, just please add changes and increase version

**Comment by nilxam** [comment]
any update? :)

**Comment by jreidinger** [comment]
looks good, feel free to merge ( if you do not have perm, then just ping me and I do it )

**Pull request merged**